src/components/modules/banners/digitalSubscriptions/DigitalSubscriptionsBanner.tsx

```diff
@@ -89,8 +91,20 @@ export const DigitalSubscriptionsBanner: React.FC<BannerProps> = ({
                 <section css={banner} data-target={bannerId}>
                     <div css={contentContainer}>
                         <div css={topLeftComponent}>
-                            <h3 css={heading}>{content?.heading}</h3>
-                            <p css={paragraph}>{content?.messageText}</p>
+                            <h3 css={heading}>
+                                <span>{content?.heading}</span>
+                                {content?.secondaryHeading && (
+                                    <span css={secondaryHeading}>{content.secondaryHeading}</span>
```

Comment:
Is this to accommodate the yellow line?

Reply:
It was, coming out now since the final copy has changed!